docker/docker-compose.yml

```diff
@@ -179,13 +187,20 @@ services:
       DEV_DSF_BPE_DB_USER_USERNAME: dic1_bpe_server_user
       DEV_DSF_BPE_DB_USER_CAMUNDA_GROUP: dic1_camunda_users
       DEV_DSF_BPE_DB_USER_CAMUNDA_USERNAME: dic1_camunda_server_user
+      DEV_DSF_BPE_SERVER_ROLECONFIG: |
+        - webbrowser_test_user:
+            thumbprint: ${WEBBROSER_TEST_USER_THUMBPRINT}
+            token-role: admin
+            dsf-role:
+              - ADMIN
       DEV_DSF_BPE_FHIR_SERVER_ORGANIZATION_IDENTIFIER_VALUE: Test_DIC1
       DEV_DSF_BPE_FHIR_SERVER_BASE_URL: https://dic1/fhir
       DEV_DSF_BPE_PROCESS_EXCLUDED: >-
```

Comment:
Consider moving the `DEV_DSF_BPE_PROCESS_EXCLUDED` config to a developer specific `docker-compose.override.yml` file and maybe add a `docker-compose.override-example.yml` file with some explanations. This would make it unnecessary to update the `mii-processes-test-setup` after mii-processes release a new version.

Reply:
done in 807b846